src/App.js

```diff
@@ -189,9 +189,11 @@ class App extends React.Component {
 
   switchCaseLoad = async (newCaseload, location) => {
```

Comment:
Definitely need a more general solution to workflows between our apps. One for later.